econml/tests/test_treatment_featurization.py

```diff
@@ -449,6 +465,8 @@ def test_featurization(self):
                         np.testing.assert_array_less(0.50, proportion_in_interval)
                         np.testing.assert_almost_equal(cme, cme_inf.point_estimate)
 
+    # def test_equal_to_outside_featuriz
```

Comment:
I assume this was something that was going to be implemented but then wasn't?  Just remove it.

```suggestion
```